tests/draftmancer/api.test.ts

```diff
@@ -29,7 +30,8 @@ describe('Publish', () => {
   beforeAll(async () => {
     app = express();
     app.use(express.json());
-    app.post('/api/draftmancer/publish', ...routes[0].handler);
+    //Need to type as our RequestHandler or Typescript gets angry
+    app.post('/api/draftmancer/publish', ...(routes[0].handler as RequestHandler[]));
```

Comment:
By using CubeCobra's extended express types in the handler Typescript complains the handlers here don't match with logger and flash params. Hence explicitly saying its our RequestHandler as well.
Probably sensible to have a simpler interface for express app in our tests so we don't have to copy/paste